docs/api/python/frozen/pyopencolorio_grouptransform.rst

```diff
@@ -69,7 +69,7 @@
 
       Overloaded function.
 
-      1. write(self: PyOpenColorIO.GroupTransform, config: PyOpenColorIO.Config = 0x1f17490 (version: 2), 0 active color spaces, 0 active displays., formatName: str, fileName: str) -> None
+      1. write(self: PyOpenColorIO.GroupTransform, config: PyOpenColorIO.Config = 0x26d9090 (version: 2), 0 active color spaces, 0 active displays., formatName: str, fileName: str) -> None
```

Comment:
Is the doc using Config.__repr__ with an empty shared pointer?
We should probably update Config.__repr__ to handle empty shared pointers.

Reply:
That's originating from the use of `EMPTY_CONFIG` being the default value in `GroupTransform.write`, yes.

Reply:
I'll update it.

Reply:
Maybe __repr__ should say "default" if self is empty (null pointer)

Reply:
It's currently printing the memory address, and in this case it's a valid, but empty, config shared ptr. I'm tweaking two things:

- Changing the config repr to print a simple `<Config name=, description=, etc.>` which will be consistent with other classes, but including the attributes currently present in the method.
- Changing GroupTransform so that the config arg defaults to `nullptr` which will be interpreted as `None` in Python, and will work with the current implementation without requiring an empty config be passed in. I do have to change the order of arguments to the write method though, since keyword args must follow positional args.

Commit coming soon.

Reply:
I tried it locally and it does not do what I expected.
This PR is great as it is.